config/prod/config.yaml

```diff
@@ -0,0 +1 @@
+artifact_bucket_name: sceptre-cloudformation-bucket-bucket-10mwvvuhlvtk9
```

Comment:
Two different buckets between develop and prod?

Reply:
Yes.